### ISSUE TITLE
Demo not allowing access to Semantics

### ISSUE THREAD

**Issue by Akamika** [watson-developer-cloud/natural-language-understanding-nodejs#13]
Text entered
Analyze selected
Default returns sentiment data
Click on Semantics button
Does not return Semantics data, just stays on the sentiment/default parse info.

**Comment by germanattanasio** [comment]
Hey @Akamika I don't understand what's your issue.

I can click on the semantic roles after clicking on analyze
<img width="816" alt="screen shot 2017-07-06 at 9 10 10 pm" src="https://user-images.githubusercontent.com/313157/27938951-93d1e7e6-628f-11e7-9cdc-46adcf0622a0.png">

**Comment by Akamika** [comment]
Hey @germanattanasio I recorded a screencast of what I experienced. 
[NLU demo issues.zip](https://github.com/watson-developer-cloud/natural-language-understanding-nodejs/files/1131017/NLU.demo.issues.zip)

No audio. It shows me entering a URL, then selecting analyze, then selecting analyze, then some of the other buttons with not info being reported. Then I switched to the default text and clicked semantic roles and it worked just fine. Maybe it's my content? The URL I was using is https://www.ibm.com/blogs/think/2017/06/science-for-social-good/
